.github/workflows/ci.yml

```diff
@@ -87,16 +86,15 @@ jobs:
       - name: Install protobuf-compiler
         run: |
          sudo apt-get install protobuf-compiler
-      - name: Run cargo-tarpaulin
-        uses: actions-rs/tarpaulin@v0.1
-        with:
-          version: ${{ env.tarpaulin-vers }}
-          args: "--avoid-cfg-tarpaulin --all-features --workspace --timeout 120 --exclude runtimes-eden nodle-parachain --exclude-files **/mock.rs **/weights.rs **/migrations.rs"
-
+      - name: Install cargo-llvm-cov
+        uses: taiki-e/install-action@cargo-llvm-cov
+      - name: Test and gather coverage
+        run: cargo llvm-cov --lcov --output-path lcov.info --all-features --workspace --exclude runtimes-eden --exclude nodle-parachain  
       - name: Upload to Codecov
         uses: codecov/codecov-action@v3.0.0
         with:
           fail_ci_if_error: false
+          files: lcov.info
```

Comment:
@simonsso looks like there is an issue here that the codecov fails to show the PR correctly.

Reply:
I will check, but it looked like clippy was unhappy when I changed the compiler to a recent nightly